services/api/database/migrations/20220908065119_initial_db.js

```diff
@@ -126,17 +126,17 @@
                 table.integer('environment');
                 table.string('severity', 300).defaultTo('');
                 table.decimal('severity_score', 1, 1).defaultTo(0);
-                table.string('identifier', 300).notNullable();
-                table.string('lagoon_service', 300).defaultTo('');
+                table.string('identifier', 100).notNullable();
+                table.string('lagoon_service', 100).defaultTo('');
```

Comment:
`lagoon_service` should be fine going to 100, pretty sure we're limited to 63 characters here anyway due to kubernetes labels and out `lagoon.sh/service` label usage. No idea what the docker-compose service name limitations are :hidethepain: